### DISCUSSION TITLE
Winpilot 2024.5.13

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#206] [Announcements]
- Clippy has been optimized again to evoke that nostalgic and authentic '90s Microsoft Clippy vibe
- Numerous subtle UI fixes and adjustments have been made
- The debloat signature of the Crapify plugin has been expanded with more "safe bloatware," now displayed in the "recommended" section for uninstallation
- Minor fixes have been made in the Setup > Privacy section
- All code changes since the version schema switch have now been finally committed, making the current codebase open source
- _For experimental purposes, a female version of Clippy dubbed MissPencil is being tested. Final name still up in the air_

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/2024.5.13'>Winpilot 2024.5.13</a>.</em>

**Comment by WDenda** [comment]
Not "MissPencil" ("Fraulein"), please "MrsPencil". Or "MrClippy" and "MrsClippy". No "DiverseClippy", although I expect there will be some demand for this, LOL. Good Luck!

**Reply by Belim** [reply]
not gonna happen, and definitely no diverse Clippy 😅. Just a little test, and the female Clippy was only offered to a handful of users. I canceled it right away.

**Comment by fudaoede** [comment]
Hello, the software is not working in Windows 10 and it is giving an error when loading as in the image
Error : the system cannot find the specific file. (exception from HRESULT: 0x80070002)
![Sem título](https://github.com/builtbybel/Winpilot/assets/170145810/ced0df72-6697-439e-9d9f-c18da71a4c62)

**Reply by Belim** [reply]
Check https://github.com/builtbybel/Winpilot/issues/204

**Reply by fudaoede** [reply]
It was already checked and I looked to see if all the files were in the folder and yes they are and it's still not the same problem

**Reply by Belim** [reply]
I meant this one https://github.com/builtbybel/Winpilot/issues/191
